src/tests/ftest/util/launch_utils.py

```diff
@@ -544,12 +603,13 @@ def _setup_test_directory(self, logger, test):
             f"sudo -n rm -fr {test_env.log_dir}",
             f"mkdir -p {test_env.log_dir}",
             f"chmod a+wrx {test_env.log_dir}",
-            f"ls -al {test_env.log_dir}",
-            f"mkdir -p {test_env.user_dir}"
         ]
         # Predefine the sub directories used to collect the files process()/_archive_files()
+        directories = [test_env.user_dir] + test_env.config_file_directories()
         for directory in TEST_RESULTS_DIRS:
-            commands.append(f"mkdir -p {test_env.log_dir}/{directory}")
+            directories.append(os.path.join(test_env.log_dir, directory))
+        commands.append(f"mkdir -p {' '.join(directories)}")
+        commands.append(f"ls -al {test_env.log_dir}")
```

Comment:
Do we really need this `ls`? The more of these we add, the more unmanageable the log files become

Reply:
This isn't additive, and it can be useful certain situations.  Maybe we can add some sort of debug parameter to the test environment for launch.py and the test framework, but in a new PR?

Reply:
In general I'm not fond of turning on some debug all the time just in case we need it for some rare edge case. But I do see how it's useful to debug issues. Maybe we do need some kind of debug/verbose flag for use in CI, but let's handle that separately. There's a lot of existing stuff in the framework already that would fall under this